processccd.rst

```diff
@@ -69,32 +72,36 @@ Entrypoint
 Butler Inputs
 =============
 
-The main method, ``run``, takes a single butler data reference for the raw input data.
+The main method, ``run``, takes a single butler data reference for the ``raw`` input data.
+
+Butler Outputs
+==============
 
 Examples
 ========
 
+The ``obs_test`` package  models a simple camera with one CCD and includes a data repository containing a few raw images (simulating three visits, two with with the `g`-band filter, and one with the `r`-band one), and some associated calibration data. Its camera consists of a single CCD whose geometry matches a subregion of a single LSST CCD.
+
 The following commands will process all raw data in obs_test's data repository. Note: be sure to specify an ``--output`` that does not already exist::
 
   setup obs_test
   setup pipe_tasks
   processCcd.py $OBS_TEST_DIR/data/input --output processCcdOut --id
 
-The data is read from the small repository in the ``obs_test`` package and written to: ``./processCcdOut`` (or whatever output you specified). Specifying ``--id`` with no values processes all data. Add the option ``--help`` to see more options.
+The data is read from the small repository in the ``obs_test`` package and output images and catalogs are written to subdirectories in: ``./processCcdOut`` (or whatever output name you specified).
+
+Specifying ``--id`` with no values processes all data.
+
+Add the option ``--help`` to see more options.
 
 
 Debugging
 =========
 
 ProcessCcdTask has no debug output, but its several subtasks do.
 
-ArgParse
-========
-
-The makeArgumentParse method creates and returns an argument parser.
-
-This override is used to delay making the data ref list until the dataset type is known; this is done in parseAndRun.
-
-ButlerInitializedTaskRunner
-===========================
+Command Line Arguments 
+======================
 
+ProcessCcdTask has all command line arguments available to a general
```

Comment:
(I believe the modification to --id is not used because IsrTask hardcodes the datasetType it takes in to 'raw' as: ccdExposure = sensorRef.get('raw')  currently)